docs/guides/admin/admin-cli.md

```diff
@@ -462,7 +462,19 @@ tsctl similarity_score
 
 #### sketch-info Get information about a sketch
 
-Displays various information about a given sketch.
+Displays detailed information about a specific sketch.
```

Comment:
One laye the sketch-info is missing: DataSource
So each timeline can have multiple data sources (files). So maybe it makes sense to also add this info to the table or a separate table? Or would you expect this detail level then with the `timeline-info` command?

Reply:
yeah I think that should go in the timeline info thing